plugin/aws/profile.go

```diff
@@ -29,7 +29,7 @@ func newProfile(ctx context.Context, name string) (*profile, error) {
 
 	// profile-specific stdin prompt
 	tokenProvider := func() (string, error) {
-		return plugin.Prompt(fmt.Sprintf("Assume ROLE MFA token code for %v", name))
+		return plugin.Prompt(fmt.Sprintf("Enter ROLE MFA token code for %v", name))
```

Comment:
This is clearer that it's a request. Might be clearer what you're enteringif if we said `Enter MFA token code for role %v`.

Reply:
Makes sense. Updated.